board/toradex/apalis-imx8/apalis-imx8.c

```diff
@@ -238,7 +239,9 @@ static void select_dt_from_module_version(void)
 			break;
 	}
 }
+#endif
 
+#ifndef CONFIG_SPL_BUILD
```

Comment:
Seems like we can drop both of these additions, which would let the `#ifndef` block continue?